python/ray/autoscaler/_private/util.py

```diff
@@ -914,6 +924,8 @@ def format_info_string(
 {separator}
 {"Total " if verbose else ""}Usage:
```

Comment:
Could you add a screenshot with verbose set to true? What is the difference between the output with "Total" and the output without "Total"?

Reply:
Can you open an issue to track the progress of the follow up and add to the umbrella issue in KubeRay repo?